### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by taehoonlee**
This PR fixes some typos: `postion`, `specifed`, and `paramter`.

### CODE REVIEW

File: lib/matplotlib/axis.py [hunk 1]
```diff
@@ -1641,7 +1641,7 @@ def _update_label_position(self, bboxes, bboxes2):
         """
         raise NotImplementedError('Derived must override')
 
-    def _update_offset_text_postion(self, bboxes, bboxes2):
+    def _update_offset_text_position(self, bboxes, bboxes2):
```
Comment (QuLogic) [inline lib/matplotlib/axis.py]: Surely there's some caller of this that also needs fixing?

Comment (taehoonlee) [inline lib/matplotlib/axis.py]: There is no caller in this repository as I examined:
```
$ grep -r "_update_offset_text_postion" . | wc -l
0
```

Comment (dstansby) [inline lib/matplotlib/axis.py]:
```
$ grep -r '_update_offset_text' lib/matplotlib
Binary file lib/matplotlib/__pycache__/axis.cpython-36.pyc matches
lib/matplotlib/axis.py:        self._update_offset_text_position(ticklabelBoxes, ticklabelBoxes2)
lib/matplotlib/axis.py:        self._update_offset_text_position(ticklabelBoxes, ticklabelBoxes2)
lib/matplotlib/axis.py:    def _update_offset_text_postion(self, bboxes, bboxes2):
lib/matplotlib/axis.py:    def _update_offset_text_position(self, bboxes, bboxes2):
lib/matplotlib/axis.py:    def _update_offset_text_position(self, bboxes, bboxes2):
```

Comment (anntzer) [inline lib/matplotlib/axis.py]: It's the abstract base method that was misspelled, so the fix is fine.

**Comment by anntzer** [comment]
thanks

**Pull request merged**